pkg/monitor/monitor/monitor_manager.go

```diff
@@ -308,14 +308,15 @@ func (mm *MonitorManager) syncTidbMonitorRbac(monitor *v1alpha1.TidbMonitor) (*c
 	}
 
 	role := getMonitorRole(monitor, policyRules)
-	role, err = mm.typedControl.CreateOrUpdateRole(monitor, role)
+	role, err = mm.typedControl.CreateOrUpdateClusterRole(monitor, role)
```

Comment:
Please do not modify here, check the original code in https://github.com/pingcap/tidb-operator/pull/3190:
1. add `spec.clusterScoped` in the CR to indicate that this TidbMonitor is for cluster scoped.
2. if `spec.clusterScoped: true`, create clusterrole and cluserrolebinding.

Reply:
`spec.clusterScoped` field is not work.  I think it is only affect tidb operator .

Reply:
I mean add `spec.clusterScoped` in the TidbMonitor CR.
BTW, for the clusterrole and clusterrolebinding, please make sure the name is unique, e.g. `ns-name`.